### ISSUE TITLE
Add a warning about keyboard navigation order

### ISSUE THREAD

**Issue by sdras** [sdras/cssgridgenerator#7]
Rachel Andrew mentioned that the way people can build with this might mess with keyboard navigation order, there should be some sort of warning: https://tink.uk/flexbox-the-keyboard-navigation-disconnect/

**Comment by sdras** [comment]
Contingent on #38

**Comment by sdras** [comment]
Fixed
